### PR TITLE
Switched to drupal_ti for travis integration

### PR DESCRIPTION
**Pull request by LKS90**
No dependencies, so I'm using the short version without the before script like in Ultimate Cron.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,77 +100,18 @@ mysql:
...
-  - echo "$(curl -fsSL https://gist.githubusercontent.com/nickveenhof/11386315/raw/b8abaf9304fe12b5cc7752d39c29c1edae8ac2e6/gistfile1.txt)" | sed -e "s,PATH,$TRAVIS_BUILD_DIR/../drupal,g" | sudo tee /etc/apache2/sites-available/default > /dev/null
-  # Set sendmail so drush doesn't throw an error during site install.
-  - echo "sendmail_path='true'" >> `php --ini | grep "Loaded Configuration" | awk '{print $4}'`
-  # Forward the errors to the syslog so we can print them
-  - echo "error_log=syslog" >> `php --ini | grep "Loaded Configuration" | awk '{print $4}'`
-  # Get latest drupal 8 core
-  - cd $TRAVIS_BUILD_DIR/..
-  - git clone --depth 1 --branch 8.0.x http://git.drupal.org/project/drupal.git
-  - cd drupal/modules
-  - git clone --depth 1 --branch 8.x-1.x https://github.com/md-systems/pathauto.git
```
Comment (Berdir) [inline .travis.yml:53]: sure about the no dependencies part? ;)

**Comment by Berdir** [comment]
Still missing one? ;)

**Comment by LKS90** [comment]
Three fails less, but still broken :(.

**Comment by Berdir** [comment]
That's OK, those were failing before. Merged!

**Pull request closed without merging**